### DISCUSSION TITLE
Class request: work shift

### DISCUSSION THREAD

**Discussion by jonathanvajda** [CommonCoreOntology/CommonCoreOntologies#386] [Announcements]
There is a wide range of use cases where employee activities are described with respect to their work shift, and this assists with coordinating resources and locating entities (such as employees, computers, vehicles, etc.). This seems useful for manufacturing, commerce, government, university, and other contexts. I think this makes a good case for it being included in the common core rather than a lower-level ontology.

I am looking for some opinions here on what this sort of entity is. It seems ambiguous whether it is:
- Act of Employment
- Temporal region
- Plan or specification for a person to perform some activities on a given temporal region
- The aggregate of activities that realize some employee role, on a given temporal region

I think we could get a recommended design pattern for this, and whether a work shift needs to be asserted above and beyond the above entities.

**Comment by cameronmore** [comment]
It seems like the first two things are what we mean when we say 'work shift'. A plan or specification for someone's work during a temporal region might be a work plan, statement of work, or task assignment ICE. I think we really mean the portion of time someone is working which occurs on planned or routine intervals. A shift worker is someone who is paid a wage for their participation in work activities at a pre-assigned time. "I was scheduled for a 2-6pm shift". We may also say that salaried employees work a 9-5pm shift, but we only say this out of convention and out of reference to the practice of shift work specifically I mentioned in the last sentence.

A work shift is the portion of time that someone is assigned or scheduled to perform tasks in accordance with the directives of a contract or entity which has deontic power to assign or schedule the performance of tasks.

**Comment by jimschoening1** [comment]
Here's one actual case:   A medical research organization needs test participants and has gotten 6000 people to fill out a long questionnaire, including, "Do you work or have you worked shifts?"

**Comment by jonathanvajda** [comment]
I have a few use cases, all fairly recently:

"Who is working a shift during this time?" (Distinction between time and
shift)

"What people are working the majority of their shifts during the same time
as another person?" (pairing coworkers)

"What times are there for someone who is working a shift and has
such-and-such certification?" (Availability for providing a service)

"Do work shifts begin or end at the same time for such-and-such percentage
of employees of job title X?" (does the change of the guard create a bottle
neck? Is it easily ascertainable to someone else that shift has changed?)
